install/kubernetes/helm/istio/test-values/values-e2e.yaml

```diff
@@ -16,7 +16,8 @@ global:
     mode: REGISTRY_ONLY
 
 prometheus:
-  scrapeInterval: 5s
+  scrapeInterval: 20s
```

Comment:
Is 20s special? It seems we intentionally lowered it for tests, but now you are raising it? If we no long need to make it a lower internal, should we just use the default 15s to test what users are likely using?

Reply:
20s seems to be doing the trick.. let me try again . without these changes